appendices/scope.md

```diff
@@ -0,0 +1,72 @@
+## Python package domain scope
+
+The following categories are the current domain areas that fall into the 
+pyOpenSci domain scope.
+
+- **Data retrieval:** Packages for accessing and downloading data from online sources. Includes wrappers for accessing APIs.
+- **Data extraction:** Packages that aid in retrieving data from unstructured sources such as text, images and PDFs.
+- **Data munging:** Tools for processing data from scientific data formats.
+- **Data deposition:** Tools for depositing data in scientific research repositories.
+- **Reproducibility:** Tools to scientists ensure that their research is reproducible. E.g. version control, automated testing, or citation tools.
+- **Geospatial:** Packages focused on the retrieval, manipulation, and analysis of spatial data.
+- **Education:** Packages to aid with instruction.
+- **Data visualization:** Packages for visualizing and analyzing data.
+
+## Package technical scope
+
+To be in technical scope for a pyOpenSci review, your package:
+
+* Should have maintenance workflows documented 
+* Should be structured in a way that someone else could contribute to it
+* Should vendor dependencies using standard approaches rather than including code from other packages within your repository. 
```

Comment:
```suggestion
* Should declare vendor dependencies using standard approaches rather than including code from other packages within your repository. 
```